packages/redis-v5/lib/shared.js

```diff
@@ -105,7 +105,7 @@ module.exports = (context, heroku) => {
 
           redis.app = db.addon.app
           redis.config_vars = db.addon.config_vars
-          const {formation, metaas_source, port, ...filteredRedis} = redis
+          const {formation, metas_source, port, ...filteredRedis} = redis
```

Comment:
???

Reply:
`metaas` is the correct spelling of an internal service.